src/components/templates/AnimateMap/components/FirebarrelWidget/FirebarrelWidget.tsx

```diff
@@ -12,11 +12,13 @@ import { useWorldUsersById } from "hooks/users";
 import { useUser } from "hooks/useUser";
 
 import LocalParticipant from "components/organisms/Room/LocalParticipant";
-import Participant from "components/organisms/Room/Participant";
 import VideoErrorModal from "components/organisms/Room/VideoErrorModal";
 
 import { Button } from "components/atoms/Button";
 
+import { WithId } from "../../../../../utils/id";
+import Participant from "../../../../organisms/Room/Participant";
```

Comment:
can you use replace all the relative paths to absolute, please? as we do in the project.
and sort them accordingly.